Collected Minor Changes - Logs and Descriptions June 2020 (#5122)
endless-sky/endless-sky: data/human/transport missions.txt

		payment
		payment 10000
		dialog "The young couple from <origin> pays you and thanks you profusely as they gather up their goods and depart. You wish them the best of luck in their new life on <planet>."

> 		log "People" "George and Samantha" `George and Samantha used to work on New Iceland before they decided to leave due to Samantha’s conditions.`

Reviewer: Ditto.

	on complete
		payment 80000
		dialog "In the past few days, Pookie has barked incessantly and thrown up several times despite being fed perfectly on schedule, and the room you've been keeping her in will likely smell like dog urine for months. You are all too happy to return her to her owner, who pays you <payment>."

> 		log "People" "Pocahontas (Pookie)" `A dog...`

Reviewer 2: It fits the moment you finish the mission, but it could be more descriptive.

> 		log "People" "Pocahontas (Pookie)" `A dog...`

Reviewer: I'm more on board with this one.

				
	on accept
		log `Visited Joe and Maria on Millrace. Life is tough, but they are doing far better now than they were on a pirate ship. Jesse has grown and the couple already has another baby to look after.`

> 		log "People" "Joe and Maria" `Two teenagers who were under the employs of pirates, the couple wanted an escape from their lives and with your aid have informed you that they are now living on Millrace in relative peace.`

Reviewer 2: This reads as if the player has helped them `inform you that...`

> 		log "People" "Joe and Maria" `Two teenagers who were under the employs of pirates, the couple wanted your aid with an escape from their lives and after a time have informed you that they are now living in relative peace where you left them on Millrace.`

Reviewer 3: ```suggestion
		log "People" "Joe and Maria" `Two teenagers who were enslaved on a pirate world, you aided the couple and their infant child Jesse to escape from their outlaw lives. You eventually helped them resettle on Millrace, and, after a time, they have informed you that they are now living there in relative peace.`
```
the original felt clunky.

		payment
		payment 10000
		dialog `You drop off the Pattersons, the farm family from <origin>, and help them to unload their wagon and cattle from your cargo hold. They thank you, and Jim pays you <payment>.`

> 		log "People" "Jim and Annette Patterson" `An unusual family who needed you to make a very unconventional trip with a sizeable number of farm animals that had to be kept happy in an enclosed trip in a spaceship.`

Reviewer 2: ```suggestion
		log "People" "Jim and Annette Patterson" `An unusual family who needed you to make a very unconventional trip with a sizeable number of farm animals that had to be kept happy in an enclosed area of your spaceship.`
```

	on decline
		payment 525894400
		"reputation: Republic" <?= -1000

> 		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the paradise planets and you decided to pocket the substantial fortune she left behind to avoid being hunted down.`

Reviewer 3: ```suggestion
		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the Paradise Planets and you decided to pocket the substantial fortune she left behind to avoid being hunted down.`
```
Aren't the Paradise Planets capatilized?

> 		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the paradise planets and you decided to pocket the substantial fortune she left behind to avioid being hunted down.`

Reviewer 2: duplicate  `decided`

Author: That's a deliberate word choice for emphasis in this case.

Reviewer 2: Fair enough. I still feel like the sentence is a bit clunky, maybe you could replace the `and` with a while, similar to the other change?

Author: While made sense on the other one because you had to go and do the returning while she started her new life. In this case though the decision making is more instantaneous. I'll have a think.

Reviewer: I'm on board with the `and -> while` change.

	on complete
		payment 1000000
		conversation
			`You spot a fleet of docked Navy ships as you come in for a landing. A small squad of troops gather around your ship after you land. You are greeted by a Navy officer upon exiting your ship and explain to him where Diana is, showing him the note she wrote.`
			`	"That is unfortunate," she responds with a troubled look on her face. "We'll contact the Howl family about this." Before letting you go, the officer hands you <payment>. "This is the reward that Mr. Howl put out for the return of his credits. Thank you for doing the right thing, Captain."`

> 		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the paradise planets and you dutifully turned in the credits to the authorities to keep the peace. You received a substantial reward for your integrity.`

Reviewer 2: ```suggestion
		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the paradise planets while you dutifully turned in the credits to the authorities to keep the peace. You received a substantial reward for your integrity.`
```

> 		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the paradise planets while you dutifully turned in the credits to the authorities to keep the peace. You received a substantial reward for your integrity.`

Reviewer 3: ```suggestion
		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the Paradise Planets while you dutifully turned in the credits to the authorities to keep the peace. You received a substantial reward for your integrity.`
```
same as above

				decline
	on complete
		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence."

> 		log "People" "Mr. Smith" `A mysterious individual who asked for 200,000 credits to receive a rich inheritance from the deceased warlord Limping Pappa, but was then proven to be a scam.`

Reviewer 2: ```suggestion
		log "People" "Mr. Smith" `A mysterious individual who asked for 200,000 credits to process a rich inheritance from the deceased warlord Limping Pappa, but was then proven to be a scam.`
```
Mr. Smith wouldn't have received the inheritance himself.

	on decline

> 		log "People" "Mr. Smith" `A mysterious individual who asked for 200,000 credits to receive a rich inheritance from the deceased warlord Limping Pappa. You presume he was a scammer and had no part in it.`

Reviewer 2: ```suggestion
		log "People" "Mr. Smith" `A mysterious individual who asked for 200,000 credits to process a rich inheritance from the deceased warlord Limping Pappa. You presume he was a scammer and had no part in it.`
```
